### PR TITLE
Feature/style updates 2019 03 28

### PR DESCRIPTION
**Pull request by nbannist**
Fixes for matching backgrounds on home and project pages with mockups as well as fixing icon badges for project cards on the dashboard.

### CODE REVIEW

**Review by kishah-iu — APPROVED** [review]
Looks good!

File: assets/css/ccore.scss [hunk 1]
```diff
@@ -7,14 +7,27 @@
...
 
 .ccore-docs {
   background-image: url("/assets/images/background-docs.png");
-  background-size: contain;
+  background-size: 300%;
 }
 
+@include mq($breakpoint-md) {
+  .ccore-home {
+    background-image: url("/assets/images/background-home.png");
```
Comment (kishah-iu) [inline assets/css/ccore.scss]: Should we switch to use `svg` at some point?

Comment (kishah-iu) [inline assets/css/ccore.scss:20]: Should we use `.svg`'s?

Comment (nbannist) [reply assets/css/ccore.scss:20]: Not for Background images. It makes Chrome (at least) lag really badly when scrolling.

Comment (kishah-iu) [reply assets/css/ccore.scss:20]: Ahh ok!

**Pull request merged**